utils/ort/src/main/kotlin/storage/HttpFileStorage.kt

```diff
@@ -36,7 +36,7 @@ import org.ossreviewtoolkit.utils.ort.OkHttpClientHelper
 import org.ossreviewtoolkit.utils.ort.execute
 
 private const val HTTP_CLIENT_CONNECT_TIMEOUT_IN_SECONDS = 30L
-private const val HTTP_CLIENT_KEEP_ALIVE_DURATION_IN_SECONDS = 5 * 60L
+private const val HTTP_CLIENT_KEEP_ALIVE_DURATION_IN_SECONDS = 1 * 60L
```

Comment:
Can you please share some details in the commit message about *why* / *how* this is actually fixing the underlying issue? It's completely unclear to me how the keep alive duration of the connection pool can have such an impact.